core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/ContainerModuleFinder.java

```diff
@@ -89,6 +89,40 @@ public void buildModule(ModuleSpec.Builder builder, ModuleLoader delegateLoader)
                         null,
                         ModuleIdentifier.create("javax.enterprise.api"), false));
 
+        builder.addDependency(
```

Comment:
Right, I'm less concerned about someone wanting to override the module directly than being able to add dependencies to a WAR and have it work